src/proxy-ui-api/build.gradle

```diff
@@ -109,6 +109,16 @@ task checkFrontAudit(type: NpmTask, dependsOn: 'npmInstall') {
     }
 }
 
+// run e2e tests vs remote url
+task e2eTest(type: NpmTask, dependsOn: 'npmInstall') {
+    inputs.dir("frontend")
+    outputs.dir("frontend")
+    args = ['run', 'test:e2e', '--', '--url=' + testTargetUrl]
```

Comment:
Now if you try to do just a regular build without running end2end tests, it fails if you don't provide a dummy testTargetUrl property: 

```
../gradlew build

> Configure project :proxy-ui-api

FAILURE: Build failed with an exception.

* Where:
Build file '/home/janne/projects/niis/xroad-ui/git/X-Road-REST-UI/src/proxy-ui-api/build.gradle' line: 116

* What went wrong:
A problem occurred evaluating project ':proxy-ui-api'.
> Could not get unknown property 'testTargetUrl' for task ':proxy-ui-api:e2eTest' 
```
Maybe some conditional logic a bit like https://stackoverflow.com/a/20694968/1469083 ? Ideally it would

1. fail the build if running e2eTest and testTargetUrl is missing (I don't think we can hardcode a sensible default without polluting the publicly shared build.gradle with some server url)
2. not fail if not running e2eTest and testTargetUrl is missing